test/stream-monitor/sm_plugin/stream_monitor.py

```diff
@@ -45,7 +51,7 @@ def configure(self, options, conf):
         super(StreamMonitorPlugin, self).configure(options,conf)
         if not self.enabled:
             return
-        if conf.options.collect_only:
+        if getattr(conf.options, 'collect_only', False):
```

Comment:
With this change, should I not see any logging info when I run_tests.py with -list option?  This is current output.
```
(eh-stu-ub16) hohene@lab17:~/sandbox-github/rackhd/stu-rackhd/test$ run_tests.py -config config-mn -stack 1 -test tests/compute/test_rackhd11_computenode_pollers.py -list
2017-01-04 15:59:53,419 22684 MainProcess stream-monitor/flogging/infra_logging.py:__init__@105 gl-main INFO this runs logging dir /emc/hohene/sandbox-github/rackhd/stu-rackhd/test/log_output/run_2017-01-04_15:59:53.d

Test Listing for: tests/compute/test_rackhd11_computenode_pollers.py
----------------------------------------------------------------------
2017-01-04 15:59:53,934 22691 MainProcess stream-monitor/flogging/infra_logging.py:__init__@105 gl-main INFO removing previous logging dir /emc/hohene/sandbox-github/rackhd/stu-rackhd/test/log_output/run_2017-01-04_15:43:52.d
2017-01-04 15:59:53,935 22691 MainProcess stream-monitor/flogging/infra_logging.py:__init__@105 gl-main INFO this runs logging dir /emc/hohene/sandbox-github/rackhd/stu-rackhd/test/log_output/run_2017-01-04_15:59:53.d
**** Using config file path: config-mn
test_1_verify_pollers (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_2_pollers_by_id (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_3_poller_headers (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_4_poller_default_cache (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_5_poller_current_data (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_6_poller_status_timestamp (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok
test_7_nodes_id_pollers (test_rackhd11_computenode_pollers.rackhd11_computenode_pollers) ... ok

----------------------------------------------------------------------
Ran 7 tests in 0.005s
```